sql/core/src/test/resources/sql-tests/results/udf/postgreSQL/udf-aggregates_part1.sql.out

```diff
@@ -101,7 +101,7 @@ struct<var_pop(CAST(CAST(udf(cast(b as string)) AS FLOAT) AS DOUBLE)):double>
 -- !query 12
 SELECT udf(var_samp(b)) FROM aggtest
 -- !query 12 schema
-struct<CAST(udf(cast(var_samp(cast(b as double)) as string)) AS DOUBLE):double>
+struct<CAST(udf(cast(var_samp(var_samp, cast(b as double)) as string)) AS DOUBLE):double>
```

Comment:
This looks weird. Looking at `Expression.toString`, I think we should override `flatArguments` in these expressions with alias to exclude the funcName parameter.

Reply:
It seems `bool_and/or` has the similar issue;
```
scala> sql("select bool_and(a) from values (true), (true), (false) t(a)").explain(true)
== Analyzed Logical Plan ==
bool_and(a): boolean
Aggregate [bool_and(bool_and, a#12) AS bool_and(a)#14]
           ^^^^^^^^^^^^^^^^^^^
+- SubqueryAlias `t`
   +- LocalRelation [a#12]
```